web/views/show.erb

```diff
@@ -52,7 +52,7 @@
 
 <div class="tab-content">
   <div class="tab-pane" id="plan">
-    <% if @plan.valid? %>
+    <% if @plan.valid? && !@plan.root_plan_step.nil? %>
```

Comment:
I would preffer `@plan.root_plan_step` over `!@plan.root_plan_step.nil?`

Reply:
It indeed looks nicer